.github/workflows/verify-manifest.yml

```diff
@@ -0,0 +1,56 @@
+name: Check Velero Manifest
+
+on:
```

Comment:
We could run this job on PR only.

on: pull_request

Reply:
Updated